src/test/java/array/Array1dTest.java

```diff
@@ -0,0 +1,207 @@
+package array;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import java.util.Optional;
+import java.util.stream.IntStream;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class Array1dTest {
+
+    @Test
+    void shouldCreateEmptyArrayByConstructor() {
+        // given
+        int size = 10;
+        // when
+        Array1d<Integer> array = new Array1d<>(size);
+        // then
+        assertNotNull(array);
+    }
+
+    @Test
+    void shouldCreateEmptyArrayByFactoryMethod() {
+        // given
+        int size = 10;
+        // when
+        Array1d<Integer> array = Array1d.of(size);
+        // then
+        assertNotNull(array);
+    }
+
+    @Test
+    void shouldVisitEmptyArray() {
+        // given
+        int size = 10;
+        Array1d<Integer> array = Array1d.of(size);
+        // expect
+        assertDoesNotThrow(() -> array.visit(i -> System.out.print(i + ", ")));
+    }
+
+    @Test
+    void shouldFillArrayWithGivenNumbers() {
+        // given
+        int start = 0;
+        int end = 9;
+        int size = 10;
+        Array1d<Integer> array = Array1d.of(size);
+        // expect
+        assertDoesNotThrow(() ->
+                IntStream.range(start, end)
+                        .forEach(i -> array.insert(
+                                Coordinates1d.of(i),
+                                10 * i)));
+    }
+
+    @Test
+    @DisplayName("Should throw an exception at second insertion into already occupied cell")
+    void shouldThrowAnException() {
```

Comment:
I think is better either use `@DisplayName` everywhere or do not use it all (preferable)

```
@Test
void shouldThrowExceptionCellIsalreadyOccupied
```
?

Reply:
Fixed d0ad4245